Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json

```diff
@@ -8676,6 +8930,634 @@
 									]
 								}
 							]
+						},
+						{
+							"name": "Required field in a New resource in Extension",
+							"item": [
+								{
+									"name": "Create busRoutes",
```

Comment:
Add a test without busRouteDirection and operatingCost values to verify those properties are marked as required